Game/graphics/mdlvisual.cpp

```diff
@@ -491,6 +491,26 @@ Tempest::Vec3 MdlVisual::displayPosition() const {
   return {0.f,0.f,0.f};
   }
 
+bool MdlVisual::boneRotationY(const char *nodeName, float& deg) const
```

Comment:
Design with return code for this function doesn't look good to me.
Can we use a `pos` matrix as-is, if bone not found?